tensorflow/core/graph/mkl_layout_pass.cc

```diff
@@ -521,33 +685,36 @@ class MklLayoutRewritePass : public GraphOptimizationPass {
   static void CopyAttrsBiasAddGrad(const Node* orign, NodeBuilder* nb);
```

Comment:
It meant original node. May be the name is not right. I changed it to orig_node.